core/domain/story_services.py

```diff
@@ -237,11 +237,13 @@ def does_story_exist_with_url_fragment(url_fragment):
 
 
 def validate_prerequisite_skills_in_story_contents(
```

Comment:
Just checking that all calls for this function were modified to sue the new parameter?

Reply:
Yes